src/modules/newPlayer.ts

```diff
@@ -0,0 +1,83 @@
+import OpenPlayerJS from 'openplayerjs';
+
+const player: any = {};
+const type = 'audio';
+
+const getMimeType = (url: string) => {
+  if (/\.m3u8/i.test(url)) {
+    return 'application/x-mpegURL';
+  } else if (/\.mpd/i.test(url)) {
+    return 'application/dash+xml';
+  } else if (/\.mp4/i.test(url)) {
+    return type === 'audio' ? 'audio/mp4' : 'video/mp4';
+  } else if (/\.mp3/i.test(url)) {
+    return 'audio/mp3';
+  } else if (/\.webm/i.test(url)) {
+    return type === 'audio' ? 'audio/webm' : 'video/webm';
+  } else if (/\.ogg/i.test(url)) {
+    return type === 'audio' ? 'audio/ogg' : 'video/ogg';
+  } else if (/\.ogv/i.test(url)) {
+    return 'video/ogg';
+  } else if (/\.oga/i.test(url)) {
+    return 'audio/ogg';
+  } else if (/\.3gp/i.test(url)) {
+    return 'audio/3gpp';
+  } else if (/\.wav/i.test(url)) {
+    return 'audio/wav';
+  } else if (/\.aac/i.test(url)) {
+    return 'audio/aac';
+  } else if (/\.flac/i.test(url)) {
+    return 'audio/flac';
+  }
+
+  // Let browser figure out if it can play it
+  return '';
+};
+
+let playerRef: any = null;
+player.init = () => {
```

Comment:
All this method should be refactor to this: 
```javascript
player.init = () => {
  if (!playerRef) {
    playerRef = new OpenPlayerJS('tarana', {
      forceNative: false
    });
    playerRef.init();
  }
  return playerRef;
};
```

Reply:
probably we are forcing the player, not to use native player it seems

Reply:
also where i can find all the options for the hls player and othe players as well ? @rafa8626

Reply:
For HLS, you can see them at https://github.com/video-dev/hls.js/blob/master/docs/API.md#fine-tuning
For FLV, you can see them at https://github.com/bilibili/flv.js/blob/master/docs/api.md#mediadatasource